Jenkinsfile

```diff
@@ -3252,6 +3252,10 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         model.max_position_embeddings=128 \
         model.encoder_seq_length=128 \
         model.data.seq_length=128 \
+        model.position_embedding_type=rope \
+        model.rotary_percent=0.5 \
```

Comment:
Here it's "rotary_percent" with default=0.5. 
Down below it's "rotary_percentage" with default=1.0.
We may want to unify them for better readability.